presidio-analyzer/presidio_analyzer/predefined_recognizers/mac_address_recognizer.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional, List, Tuple
+
+from presidio_analyzer import Pattern, PatternRecognizer
+
+
+class MACAddressRecognizer(PatternRecognizer):
+    """
+    Recognizes MAC Address using Regex pattern.
+
+    :param patterns: List of patterns to be used by this recognizer
+    :param context: List of context words to increase confidence in detection
+    :param supported_language: Language this recognizer supports
+    :param supported_entity: The entity this recognizer can detect
+    """
+
+    
+    PATTERNS = [
+        Pattern(
+            "MAC Address (high)",
+            r"\b[0-9A-Fa-f]{12}\b",
```

Comment:
The pattern should reflect something like
00:1b:63:84:45:e6 or as 00-1B-63-84-45-E6

where `every 2 letters` are interleaved by either a : or -`

Reply:
this handled internally using "replacement_pairs".